README.md

```diff
@@ -361,7 +361,7 @@ Test Group Name| Description
 `cnb` | Tests our ability to use cloud native buildpacks.
 `detect` | Tests the ability of the platform to detect the correct buildpack for compiling an application if no buildpack is explicitly specified.
 `docker`| Tests our ability to run docker containers on Diego and that we handle docker metadata correctly.
-`file-based service bindings`| Tests file-based service bindings for a buildpack app, a CNB app and a Docker app.
+`file-based service bindings`| Tests file-based service bindings for a buildpack app, a CNB app and a Docker app. This test group being run against 2 different feature flags on 2 different stacks(windows,cflinuxfs4). For more detail about the feauture flags check [RFC0030](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0030-add-support-for-file-based-service-binding.md) 
```

Comment:
We shouldn't hard-code the "cflinuxfs4" stack name here. Just mention that the test runs on Linux and Windows.